CodeQL.yml

```diff
@@ -0,0 +1,6 @@
+path_classifiers:
```

Comment:
Does this file have to be in the root of the repo? If so, I'd like to keep the root clean of these.

Reply:
Yes, based on their guideline [here](https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/bugs/generated-library-code). The file can be named `CodeQL.yml` or `.CodeQL.yml` at the root of the repo

Reply:
The docs make it sound like there's a way to configure the location of the CodeQL.yml file but I couldn't figure out how.

It also looks like using the `Codeql.ExcludePathPatterns` variable could be another option: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/troubleshooting/bugs/generated-library-code#alternative-to-codeqlyml

Reply:
@weshaggard We can look at the alternative and add exclusion to the pipelines instead. What do you think about that approach?

Reply:
@weshaggard I update the `1es-redirect.yml` file with the exclusion properties. I check and do not see OneBranch template or CodeQL3000/ Azure Native pipeline apply to our repo, but let me know if I miss something.